tools/c7n_gcp/c7n_gcp/actions/core.py

```diff
@@ -32,7 +32,7 @@ class MethodAction(Action):
     """
 
     # method we'll be invoking
-    method_spec = ()
+    method_spec = {}
 
```

Comment:
mutable class vars are a bit of a red herring, in that they rarely signal usage which has caveats.